bookea-frontend/src/modules/user/pages/UserProfilePage.module.css

```
@@ -0,0 +1,97 @@
+.pageContainer {
+    max-width: 800px;
+    margin: 2rem auto;
+    padding: 0 1.5rem;
+    font-family: 'Inter', sans-serif;
```

Stylelint (17.6.0)

[error] Unexpected quotes around "Inter" (font-family-name-quotes)

(font-family-name-quotes)